src/errors/meilisearch-api-error.ts

```diff
@@ -25,9 +25,17 @@ const MeiliSearchApiError: MSApiErrorConstructor = class
     this.errorLink = error.errorLink
     this.message = error.message
     this.httpStatus = status
-
-    if (Error.captureStackTrace) {
-      Error.captureStackTrace(this, MeiliSearchApiError)
+    if (error.stack) {
+      this.stack = error.stack;
+      this.stack = this.stack?.replace(/(TypeError|FetchError)/, this.name)
+      this.stack = this.stack?.replace(
+        'Failed to fetch',
+        `request to ${error.errorLink} failed, reason: connect ECONNREFUSED`
+      )
```

Comment:
If this error occurs it will go to `meilisearch-communication-error` and not here. So this is redundant. 
It is not a ECONNREFUSED, it is a 404 `Not found`. 
You'll have to catch that one and transform it

Reply:
How to reproduce this error?

Reply:
Done

Reply:
@bidoubiwa  as per the issue description it says we need to make changes in the MeiliSearchApiError 

**Add information in MeiliSearchApiError when http error is raised #931**